akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -576,29 +566,31 @@ object HttpEntity {
   object LastChunk extends LastChunk("", Nil)
 
   /**
-   * Turns the given source into one that respects the `withSizeLimit` calls when used as a parameter
-   * to entity constructors.
+   * Deprecated: no-op, not explicitly needed any more.
    */
+  @deprecated("Not needed explicitly any more. ", "10.1.4")
```

Comment:
10.1.4 already released so should be 10.1.5, right?

Reply:
CONTRIBUTING.md says "src/main/mima-filters/<last-released-version>.backwards.excludes" but this always confuses me so I'm not entirely confident...

Reply:
In contrast to MiMa :laughing: